lib/bank_account/account.rb

```diff
@@ -0,0 +1,26 @@
+module Bank
+  class Account
+    attr_accessor :id, :balance
+
+    def initialize(id, initial_balance)
+      # raises error for negative initial balance
+      raise ArgumentError.new "Invalid initial balance" unless initial_balance > 0
+      @id = id
```

Comment:
It might be nice to isolate the error raising into its own method that either returns `initial_balance` or raises the error. For instance, maybe something like this:

```ruby
def initialize(id, initial_balance)
  @id = id
  @balance = set_initial_balance(initial_balance, 0)
end

def set_initial_balance(balance, min_amount)
  if balance > min_amount
    balance
  else
    raise ArgumentError.new "Invalid initial balance"
  end
end
```
Having a method like `set_initial_balance` might come in handy in your other classes where you're repeating similar behavior.